esp_encrypted_img/src/esp_encrypted_img.c

```diff
@@ -452,6 +450,7 @@ esp_err_t esp_encrypted_img_decrypt_end(esp_decrypt_handle_t ctx)
 exit:
     mbedtls_gcm_free(&handle->gcm_ctx);
     free(handle->cache_buf);
+    free(handle->rsa_pem);

```

Comment:
```suggestion
    if (handle->rsa_pem) {
        free(handle->rsa_pem);
    }

Reply:
Freeing a `NULL` pointer is a no-op, I make sure to set the pointer NULL when it's free'd early

Reply:
`handle->rsa_pem` will be freed after decrypting gcm key. Freeing it here again might cause an error.